### PR TITLE
docs: ai podcast guide part 2

### PR DESCRIPTION
**Pull request by jyecusch**

### CODE REVIEW

File: docs/guides/python/ai-podcast-part-2.mdx [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+model.zip
+```
+
+Finally, we need to tell Nitric to use these files to create the script service. We can do this by updating the `nitric.yaml` file:
+
+```yaml title:nitric.yaml
+name: ai-podcast
+services:
+  - match: services/*.py
+    start: uv run watchmedo auto-restart -p *.py --no-restart-on-command-exit -R python -- -u $SERVICE_PATH
```
Comment (davemooreuws) [inline docs/guides/python/ai-podcast-part-2.mdx:382]:
```suggestion
    start: uv run watchmedo auto-restart -p *.py --no-restart-on-command-exit -R uv run $SERVICE_PATH
```

File: docs/guides/python/ai-podcast-part-2.mdx [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+    runtime: python
+batch-services:
+  # !diff -
+  - match: batches/*.py
+  # !diff(1:6) +
+  - match: batches/podcast.py
+    start: uv run watchmedo auto-restart -p *.py --no-restart-on-command-exit -R python -- -u $SERVICE_PATH
```
Comment (davemooreuws) [inline docs/guides/python/ai-podcast-part-2.mdx:389]:
```suggestion
    start: uv run watchmedo auto-restart -p *.py --no-restart-on-command-exit -R uv run $SERVICE_PATH
```

File: docs/guides/python/ai-podcast-part-2.mdx [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+    runtime: torch
+  - match: batches/script.py
+    start: uv run watchmedo auto-restart -p *.py --no-restart-on-command-exit -R python -- -u $SERVICE_PATH
```
Comment (davemooreuws) [inline docs/guides/python/ai-podcast-part-2.mdx:392]:
```suggestion
    start: uv run watchmedo auto-restart -p *.py --no-restart-on-command-exit -R uv run $SERVICE_PATH
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]
Some changes to the yaml for windows

File: docs/guides/python/ai-podcast-part-2.mdx [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+@main_api.post("/podcast/:title")
+async def submit_script(ctx: HttpContext):
+  title = ctx.req.params["title"]
+  body = ctx.req.data
+
+  if body is None:
+    ctx.res.status = 400
+    return
+
+  await generate_podcast.submit({"title": title, "prompt": body.decode('utf-8')})
```
Comment (davemooreuws) [inline docs/guides/python/ai-podcast-part-2.mdx:310]: need to define generate_podcast above as a diff +

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Comment by davemooreuws** [comment]
Noticed there isn't any screenshots or steps for running it locally like there was in part 1. Is this something we want to add? @jyecusch

**Review by tjholm — DISMISSED** [review]

File: docs/guides/python/ai-podcast-part-2.mdx [hunk 5]
```diff
@@ -0,0 +1,451 @@
...
+model.zip
+```
+
+## Update our .env file
+
+We'll change some of the environment variables in our `.env` file to factor in the extra init time it will take to start our LLM job:
+
+```sh title:.env
+PYTHONPATH=.
+# diff +
```
Comment (davemooreuws) [inline docs/guides/python/ai-podcast-part-2.mdx:391]:
```suggestion
# !diff +
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

**Review by tjholm — APPROVED** [review]

**Pull request merged**